client/components/index.jsx

```diff
@@ -1,16 +1,75 @@
+/* eslint-disable */
 import React from 'react';
 import ReactDOM from 'react-dom';
+import axios from 'axios';
+import styled from 'styled-components';
+import MarketOrder from './marketOrder.jsx'
+import Wrapper from './styles/mainWrapper/wrapper';
+import InputWrapper from './styles/inputWrapper/inputWrapper';
+import Span from './styles/Span/span';
+import GlobalStyle from './styles/globalStyle';
+
+const EstimateWrapper = styled(InputWrapper)`
+    border-top: 1px solid black;
+`
 
 class App extends React.Component {
     constructor() {
         super()
 
-        this.state = {};
+        this.state = {
+            ticker: "",
+            bp: 0,
+            shares: 0,
+            price: 0,
+            estimate: 0,
+            type: 'Buy',
+            tab: 'Market Order'
+        };
```

Comment:
You may want to store share state in one location so that it can be updated all at once